doc/release/1.10.0-notes.rst

```diff
@@ -73,6 +73,12 @@ given period and one end point is added to each extremity of *xp* in order to
 close the previous and the next period cycles, resulting in the correct
 interpolation behavior.
 
+*np.pad* supports more input types for ``pad_width`` and ``constant_values``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``constant_values`` parameters now accepts NumPy arrays and float values.
```

Comment:
Explanation isn't very clear, assumes too much knowledge of the original function. But lets wait until the rest is settled.